packages/flutter_tools/lib/src/commands/create_base.dart

```diff
@@ -100,14 +100,15 @@ abstract class CreateBase extends FlutterCommand {
       abbr: 'i',
       defaultsTo: 'swift',
       allowed: <String>['objc', 'swift'],
-      help: 'The language to use for iOS-specific code, either Objective-C (legacy) or Swift (recommended).'
+      help: '(deprecated) The language to use for iOS-specific code, either Swift (recommended) or Objective-C (legacy).',
+      hide: !verboseHelp,
     );
     argParser.addOption(
       'android-language',
       abbr: 'a',
       defaultsTo: 'kotlin',
       allowed: <String>['java', 'kotlin'],
-      help: 'The language to use for Android-specific code, either Java (legacy) or Kotlin (recommended).',
+      help: 'The language to use for Android-specific code, either Kotlin (recommended) or Java (legacy).',
```

Comment:
I don't know enough abt this area but does this section need `hide: !verboseHelp,` too?

Reply:
`--android-language` isn't being deprecated, and not in this PR.  I just changed the order to match the iOS one, and because putting the recommended one first is a good suggestion.